pkg/storagepolicy/sdkstoragepolicy.go

```diff
@@ -39,9 +40,9 @@ type SdkPolicyManager struct {
 }
 
 const (
-	policyPrefix = "/storage/policy"
+	policyPrefix = "storage/policy"
 	policyPath   = "/policies"
-	enforcePath  = "/storage/policy/enforce"
+	defaultPath  = "storage/policy/enforce"
```

Comment:
Just checking. Looks like you remove the leading `/`

Reply:
yes I observed in etcd2 and etcd3 `/ ` is treated differently 
for eg. 
in etcd3 this is allowed -
```
root_path/clusterid//storage/policy
root_path/clusterid/storage/policy
root_path/clusterid/cluster/node
root_path/clusterid//cluster/role/...
```
so keeping inline to avoid unexpected errors